capstone-frontend/src/BookSearchTile.test.js

```diff
@@ -42,16 +42,22 @@ test('renders book search tile test', () => {
   const bookListButton = document.getElementById("dropdown-list-add");
   expect(bookListButton).toBeInTheDocument();
   expect(bookListButton.innerHTML).toEqual("Add to List");
-  
 
+  await act(async () => {bookListButton.click()});
```

Comment:
I'm not familiar with `act`, but from the documentation it looks to me like you might not need to use `async`/`await` here, as long as you wrap any actions that have downstream UI effects you want to assert on in a call to `act`:
https://reactjs.org/docs/test-utils.html#act
https://reactjs.org/docs/testing-recipes.html#act

Reply:
Discussed offline and couldn't find a way to avoid the use of async and await here.